consensus/fork/galactica_test.go

```diff
@@ -455,7 +455,7 @@ func TestValidateGalacticaTxFee(t *testing.T) {
 			tx:           tx.NewBuilder(tx.TypeLegacy).GasPriceCoef(255).Build(),
 			baseFee:      defaultBaseFee,
 			baseGasPrice: new(big.Int).Sub(defaultBaseFee, common.Big1),
-			wantErr:      errors.New("max fee per gas is less than block base fee"),
+			wantErr:      errors.New("base fee too high for legacy tx, use dynamic fee tx or retry later"),
```

Comment:
Here we could replace the `wantErr: errors.New(...` with `wantErr: errBaseFeeTooHighForLegacyTx` directly. Also on line 465 with the dyn error msg

Reply:
Done, thanks!